v3.go

```diff
@@ -27,7 +27,6 @@ var logger = zerolog.New(os.Stdout).With().Timestamp().Logger().Output(zerolog.C
 var Version string
 
 type Opts struct {
-	Debug                bool
```

Comment:
An integration test needs to be fixed up to account for this change, see v3_test.go